apisix/plugins/ext-plugin/init.lua

```diff
@@ -700,6 +701,9 @@ function _M.communicate(conf, ctx, plugin_name)
 
         if not core.string.find(err, "conf token not found") then
             core.log.error(err)
+            if conf.allow_degradation then
+                return
```

Comment:
Let's add a logging message here? So we can detect the degradation situation, also, we can enrich our test cases.

Reply:
have changed